### PR TITLE
Fix typo, add feh to DE mode, update README

### PR DESCRIPTION
**Pull request by ChiDal**
I added feh to DE mode after gsettings so that it works in others too

### CODE REVIEW

File: pacwall.sh [hunk 1]
```diff
@@ -128,7 +128,10 @@ set_wallpaper() {
         </wallpapers>" \
             > "${XDG_DATA_HOME}/gnome-background-properties/pacwall${BACKGROUND}.xml"
 
-        gsettings set org.gnome.desktop.background picture-uri "${XDGOUT}" || true
+        feh --bg-center --no-fehbg --image-bg "$BACKGROUND" "${XDGOUT}" \
+            2> /dev/null && echo 'Using feh to set the wallpaper'
```
Comment (Kharacternyk) [inline pacwall.sh]: Could you add `hsetroot` backend as well?

File: README.rst [hunk 2]
```diff
@@ -42,6 +42,7 @@ Use `-D` to enable desktop environment integration (KDE Plasma, GNOME, ...). You
 DE integration requires ``imagemagick`` and ``xorg-xdpyinfo`` (If you are on Wayland you need to specify the screen size manually like this: ``./pacwall.sh -S 1920x1200``).
 
 **WARNING**: Setting a wallpaper in GNOME and the derivatives isn't possible with ``feh`` and ``hsetroot``. ``-D`` is required.
+             (If you don't know your DE, it is probably GNOME)
```
Comment (Kharacternyk) [inline README.rst]: GitHub renders this a bit weird. Whitespace is significant in reStructuredText, maybe this is the problem? Could you join these two lines?

**Comment by ChiDal** [comment]
Done

**Comment by Kharacternyk** [comment]
Thanks!

**Pull request merged**